python-sdk/src/astro/files/types/base.py

```diff
@@ -45,3 +45,6 @@ def __repr__(self) -> str:
 
     def __eq__(self, other):
         return self.name == other.name
+
+    def __hash__(self):  # pragma: no cover
+        return hash(self.name)
```

Comment:
Can we please have a simple test for this? Either don't add the comment or add a test. But we shouldn't ever put `no cover` anywhere except for `if TYPE_CHECKING`. Everything we can test, we should test.